tests/onebit/test_mpi_backend.py

```diff
@@ -4,26 +4,26 @@
 import torch.distributed as dist
 import numpy as np
 import deepspeed
-from deepspeed.runtime.fp16.onebit_adam import OnebitAdam
+
+from deepspeed.runtime.comm.mpi import MpiBackend
 
 comm = MPI.COMM_WORLD
 size = comm.Get_size()
 rank = comm.Get_rank()
 
 #TODO: Detect the hostname we are running on automatically
 torch.distributed.init_process_group(backend='nccl',
-                                     init_method='tcp://worker-1:2245',
+                                     init_method='tcp://worker-0:2245',
```

Comment:
side note, you should be able to replace this with `deepspeed.init_distributed` (https://deepspeed.readthedocs.io/en/latest/initialize.html#distributed-initialization) it will use mpi4py to properly initialize torch.distributed.

Reply:
Resolved, thanks!